cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/core/service/worker/impl/CoordinateCleanerImpl.java

```diff
@@ -0,0 +1,243 @@
+                                    coords.clear();
+                                }
+                                if(coords.size() == 2) {
+                                    float d = Float.parseFloat(coords.get(0));
+                                    float m = Float.parseFloat(coords.get(1));
+                                    double dd = Math.signum(d) * (Math.abs(d) + (m / 60.0));
+                                    formattedCoords.add(dd);
+                                    coords.clear();
+                                }
+                                if(coords.size() == 1) {
+                                    Double doubleCoord = Double.parseDouble(coords.get(0));
+                                    formattedCoords.add(doubleCoord);
+                                    coords.clear();
+                                }
+                               
+                            } else if(s.equals("S") || s.equals("W")) {
+                                // handle negative coords
+                                if(coords.size() == 3) {
+                                    float d = Float.parseFloat(coords.get(0));
+                                    float m = Float.parseFloat(coords.get(1));
+                                    float se = Float.parseFloat(coords.get(2));
+                                    double dd = 0;
+                                    if(d == 0) {
+                                        dd = ((m / 60.0) + (se / 3600.0));
+                                    } else {
+                                        dd = Math.signum(d) * (Math.abs(d) + (m / 60.0) + (se / 3600.0));
+                                    }
+                                    dd = dd *-1;
+                                    formattedCoords.add(dd);
+                                    coords.clear();
+                                }
+                                if(coords.size() == 2) {
+                                    float d = Float.parseFloat(coords.get(0));
+                                    float m = Float.parseFloat(coords.get(1));
+                                    double dd = Math.signum(d) * (Math.abs(d) + (m / 60.0));
+                                    dd = dd *-1;
+                                    formattedCoords.add(dd);
+                                    coords.clear();
+                                }
+                                if(coords.size() == 1) {                
+                                    Double doubleCoord = Double.parseDouble(coords.get(0)) * -1;
+                                    formattedCoords.add(doubleCoord);
+                                    coords.clear();
+                                }
+                            }
+                       }
+                       CleanedCoordinatesImpl cleanedCoords = new CleanedCoordinatesImpl();
+                       cleanedCoords.setType("Point");
```

Comment:
should not be hard-coded

Reply:
What is the java equivalent of importing from your settings file? Also if the type in this case is always going to be a "Point" whats wrong with it being hard coded? Coordinates could be polygons but this function does not currently handle polygons as there is no need.

Reply:
What if we use it somewhere else in the code and then it gets renamed for some reason? Then you would have to find all the places where "Point" is hard-coded. Or if this is just a way to distinguish types, you would have to remember that you called it "Point" and not "point" or "POINT" in other places of the code.

Either put it in an enum or interface constant or in the config file and read it from there. If it is mainly to distinguish coordinates by types in the system (not user facing), I would probably use an enum.